en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,81 @@
         
-2. Click the graph on a high point.
-3. Observe the logs in the logs panel.
-4. Note multiple logs are mentioning a connection error with the `hr-service`.
+2. Click on a data point where there is a spike and observe the log entries in the **Logs** pane.
  
-    ![Analyzing the throughput graph](../assets/img/observability/throughput-graph-analysis.png){.cInlineImage-full}
- 
-#### Analyze the latency graph
+    ![Analyze the throughput graph](../assets/img/observability/throughput-graph-analysis.png){.cInlineImage-full}
 
-The latency graph shows the latencies of each request.
+   Here, the log displays multiple entries indicating an error in connecting to the `hr-service`.
 
-With the sample data, you can observe spikes in the latency graph. You can identify the errors that caused the spikes and the exact API invocation(s) that caused the errors by analyzing the latency graph.
+ 
+### Analyze the latency graph
 
-![Analyzing the latency graph](../assets/img/observability/latency-graph-analysis.png){.cInlineImage-full}
+The latency graph depicts the latency of requests over a selected time interval. You can analyze the spikes in the latency graph to identify the API invocations that result in error status.
 
-Follow the steps below.
+Follow this procedure to analyze the latency graph:
 
-1. Hover over the latency graph and click on a high point.
+1. Go to the latency graph under the **Throughput & Latency** tab and click on a data point where there is a spike. This displays the latency, start time, and status of the corresponding requests just above the low-code diagram.
    
     !!!tip
-        You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+        If you want to drill down to view details at a higher granularity, drag the cursor over a time range to view the graph for the selected period.
         
-2. Observe the latencies of each request listed in the low-code diagram.
-3. Click on one request shown on the low-code diagram (latencies). You can observe the status code for that request and thereby identify the exact API invocation that caused any error(s).
+2. Click on a latency value to observe the status code for that request.
+ 
+    ![Analyze the latency graph](../assets/img/observability/latency-graph-analysis.png){.cInlineImage-full}
 
-### Step 2: Drill into the analytics
+   Here, the status code is `500`, which means the request resulted in an error state.
 
-![Diagnostic View](../assets/img/observability/diagnostics-view.png){.cInlineImage-full}
+### Analyze diagnostic data
 
-Follow the steps below to learn how to drill down and confirm the root cause for the anomalies we detected in the **Latency and Throughput** graphs.
+The **Diagnostic View** allows you to simultaneously analyze observability data to identify possible root causes for anomalies of a service.
```

Comment:
The diagnostic view is a more of a drill-down

Reply:
Rephrased.